### DISCUSSION TITLE
Updates are erasing settings

### DISCUSSION THREAD

**Discussion by webby2001** [brianpetro/obsidian-smart-connections#435] [General]
Each update seems to be resetting my settings for note and block models. Any way to stop that?

**Comment by nomadphase** [comment]
Yes, this has been quite frustrating and I'd like to see some kind of opt-in approach to re-embedding the vault.

I am happy overall that the quality of embeddings and responses appears markedly improved

**Reply by brianpetro** [reply]
@nomadphase 

> I am happy overall that the quality of embeddings and responses appears markedly improved

😁

> opt-in approach to re-embedding the vault

Currently, there is a threshold where if the number of new embeddings is >300 then Smart Connections should require hitting "Start Embedding" in a notification prior to the embedding process.

I'm thinking maybe that threshold should be configurable. What do you think?

Also, if it's automatically embedding >300, let me know, and under what circumstances, because it shouldn't be.

Thanks for the help 😊

🌴

**Comment by brianpetro** [comment]
@webby2001 @nomadphase does this happen when you turn on/off the plugin?

Thanks for your help in figuring this one out!

🌴

**Reply by webby2001** [reply]
Didn't seem to? Just tried that on my Mac (14.2.1)

**Reply by brianpetro** [reply]
That's good.

I'm thinking the change might've not necessarily been a bug, but because a change I introduced (basically, renaming the settings).

Let me know if it happens again, though. Because if so, then there might be something I'm missing. 

Thanks for your help!

🌴
